src/platform/index.js

```diff
@@ -0,0 +1,79 @@
+import { filter, find, forEach } from 'min-dash';
```

Comment:
I'm asking myself whether `platform` is the best term here. I know we are not super aligned across the libraries, but maybe we should at least use `camunda-platform` (as in the properties panel)?